airflow/jobs.py

```diff
@@ -691,10 +694,15 @@ def manage_slas(self, dag, session=None):
                         if email not in emails:
                             emails.append(email)
             if emails and len(slas):
-                send_email(
-                    emails,
-                    "[airflow] SLA miss on DAG=" + dag.dag_id,
-                    email_content)
+                try:
+                    send_email(
+                       emails,
+                       "[airflow] SLA miss on DAG=" + dag.dag_id,
+                       email_content)
+                    email_sent = True
```

Comment:
Why set this variable here? It is also set to `True` some lines below.

Reply:
Right, I missed that.